### DISCUSSION TITLE
Data not stored when calling saveQuietly

### DISCUSSION THREAD

**Discussion by boonstoppel** [clickbar/laravel-magellan#45] [Q&A]
This does not store any data:

```
$port->location = Point::makeGeodetic(55, 11);
$port->saveQuietly();
```

only this does:
```
$port->location = Point::makeGeodetic(55, 11);
$port->save();
```

Is that on purpose / a limitation by design or can that be fixed?

**Comment by saibotk** [comment]
We do not know of any kind of limitation like that but i will look into this later, maybe i can quickly fix it if possible, thanks for reporting this!

**Comment by boonstoppel** [comment]
Actually, sorry, this was a race condition on my end and `saveQuietly` works as intended.
